### PR TITLE
Add config builder

### PR DESCRIPTION
**Pull request by ansermino**
Copies config builder from ChainBridge repo.

- Fixes an issue where we weren't providing relayer addresses for substrate chains

### CODE REVIEW

**Comment by steviezhang** [comment]
Just retriggered the build on this, will review in a bit

File: cfgBuilder/main.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+)
+
+var app = cli.NewApp()
+
+func init() {
+	app.Action = run
+	app.Copyright = "Copyright 2019 ChainSafe Systems Authors"
+	app.Name = "cfgBuilder"
+	app.Usage = "cfgBuilder [source] [destination]"
+	app.Author = "ChainSafe Systems 2019"
```
Comment (steviezhang) [inline cfgBuilder/main.go:25]: [Author field has been depreciated](https://pkg.go.dev/github.com/urfave/cli?tab=doc#App). Should use `app.Authors` and assign an Author struct to it.

```
type Author struct {
	Name  string // The Authors name
	Email string // The Authors email
}
```

Comment (ansermino) [reply cfgBuilder/main.go:25]: Good catch! The error in CI is actually an issue with go mods, but it does look like we should update the CLI at some point https://pkg.go.dev/mod/github.com/urfave/cli/v2@v2.2.0

File: cfgBuilder/main.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	app.EnableBashCompletion = true
+}
+
+func run(ctx *cli.Context) error {
```
Comment (steviezhang) [inline cfgBuilder/main.go:29]: Seems like `func(*cli.Context) {}` is considered deprecated right now too, and docs mention support will be removed for it in a later version. Do we want to get ahead of that?

Comment (ansermino) [reply cfgBuilder/main.go:29]: Not a priority here, opened issue to update to v2 over in ChainBridge

**Pull request merged**